### ISSUE TITLE
howto add username and password?

### ISSUE THREAD

**Issue by githubbbie** [DeepSpaceHarbor/hexo-bridge#184]
I would like to track username and password for users and leave a trail/log of edits made using the system.

Any good ideas howto implement this on this wonderful system?

**Comment by DeepSpaceHarbor** [comment]
The plugin listens on 2 routes: `/api/*` for the admin API routes, and  `/bridge/*` route, which is used for the frontend client.  In the past I've recommended to others to simply put those routes as protected directories. If you're using apache server, that would mean editing the `.htaccess` file, or something equivalent for the other servers.

**Comment by githubbbie** [comment]
Good idea, will try this and report back on how did it so everyone benefits.
